sdk/identity/identity/src/index.ts

```diff
@@ -50,7 +50,7 @@ export { logger } from "./util/logging";
 export { AzureAuthorityHosts } from "./constants";
 
 /**
- * Returns a new instance of the {@link DefaultAzureCredential}.
+ * Returns a new instance of the {@link @azure/identity.DefaultAzureCredential}.
```

Comment:
While we may have considered this for links to symbols from an external package, this is an overhead for a symbol in the same package.

Reply:
This is a non starter for symbols in the same package. I'll follow up offline with @DavidCBerry13 on this